README.md

```diff
@@ -1,100 +1,18 @@
-[![GraalVM](.github/assets/logo_320x64.svg)][website]
+# GraalVM Conclave fork
 
-[![GraalVM downloads][badge-dl]][downloads] [![GraalVM docs][badge-docs]][docs] [![GraalVM on Slack][badge-slack]][slack] [![GraalVM on Twitter][badge-twitter]][twitter] [![GraalVM on YouTube][badge-yt]][youtube]  [![GraalVM Gate][badge-gate]][gate] [![License][badge-license]](#license)
+This is a fork of the [Oracle GraalVM repo](https://github.com/oracle/graal) with modifications to get GraalVM 
+working with [Conclave](https://github.com/R3Conclave/conclave-sdk) inside an Intel SGX enclave.
 
-GraalVM is a high-performance JDK distribution designed to accelerate the execution of applications written in Java and other JVM languages along with support for JavaScript, Ruby, Python, and a number of other popular languages.
+Unlike upstream, this repo's default branch is `conclave/master` and is initially based off the `vm-22.0.0.2`
```

Comment:
What do we mean by `upstream` here? Not clear to me as a first time reader. Maybe you meant the repo root?